frontend/www/js/omegaup/arena/ranking.test.ts

```diff
@@ -0,0 +1,96 @@
+jest.mock('../../../third_party/js/diff_match_patch.js');
+
+import { types } from '../api_types';
+import { onRankingChanged, updateProblemScore } from './ranking';
+
+describe('ranking', () => {
+  const scoreboard = {
+    problems: [
+      {
+        alias: 'problem_alias',
+        order: 1,
+      },
+      {
+        alias: 'problem_alias_2',
+        order: 2,
+      },
+    ],
+    ranking: [
+      {
+        classname: 'user-rank-unranked',
+        username: 'omegaUp',
+        country: 'MX',
+        is_invited: true,
+        problems: [
+          {
+            alias: 'problem_alias',
+            points: 100,
+            runs: 1,
+            percent: 0,
+            penalty: 3,
+          },
+          {
+            alias: 'problem_alias_2',
+            points: 100,
+            runs: 3,
+            percent: 0,
+            penalty: 5,
+          },
+        ],
+        total: {
+          points: 0,
+          penalty: 0,
+        },
+      },
+    ],
+    start_time: new Date(),
+    time: new Date(),
+    title: 'contest',
+  } as types.Scoreboard;
+  const navbarProblems = [
+    {
+      acceptsSubmissions: true,
+      alias: 'problem_alias',
+      bestScore: 100,
+      hasRuns: true,
+      maxScore: 100,
+      text: 'A. Problem',
+    },
+    {
+      acceptsSubmissions: true,
+      alias: 'problem_alias_2',
+      bestScore: 80,
+      hasRuns: true,
+      maxScore: 100,
+      text: 'B. Problem 2',
+    },
+  ] as types.NavbarProblemsetProblem[];
```

Comment:
```suggestion
  ];
```

si es una declaración, no se usa `as`: eso está reservado para expresiones anónimas.

Reply:
Ah, cierto.